src/VisualStudio/Core/Def/ColorSchemes/VisualStudio2017.xml

```diff
@@ -529,8 +529,8 @@
             <Color Name="label name">
             </Color>
             <Color Name="inline parameter name hints">
-                <Foreground Type="CT_RAW" Source="FF959493" />
                 <Background Type="CT_RAW" Source="FF2D2C2C" />
+                <Foreground Type="CT_RAW" Source="FF959493" />
```

Comment:
The tool that generates pkgdef files from xml requires Background elements to be defined first.